regexes/device/mobiles.yml

```diff
@@ -1732,7 +1751,7 @@ Vizio:
 
 # GOCLEVER
 GOCLEVER:
-  regex: 'GOCLEVER|QUANTUM|ARIES|INSIGNIA'
+  regex: 'GOCLEVER|QUANTUM|ARIES|INSIGNIA|ORION_|ELIPSO'
```

Comment:
Matching without `_` wrongly detects following user agent from other tests:

`Linux; U; Android 2.3.6; it-it; NGM Orion Build/GRK39F) AppleWebKit/533.1 (KHTML, like Gecko) Version/4.0 Mobile Safari/533.1`

I'd rather leave it as it's now as I don't have idea how to target this case